contracts/ERC20Safe.sol

```diff
@@ -52,6 +53,18 @@ contract ERC20Safe is BridgeRole, Pausable {
     event ERC20Deposit(uint112 depositNonce, uint112 batchId);
     event ERC20SCDeposit(uint112 indexed batchId, uint112 depositNonce, string callData);
```

Comment:
if you compile the sol contracts, this event will have the `CallData` field defined as string not []byte. @dragos-rebegea did a recent change in which we changed this field from string to []byte to avoid double hex encoding. How do we fix this?

Reply:
merged latest version, should be good now